components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint.util/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/util/AuthenticationEndpointUtil.java

```diff
@@ -405,26 +403,27 @@ public static String sendGetRequest(String backendURL) {
      * @return Extracted http response content.
      * @throws IOException if there is an error while extracting the response content.
      */
-    private static StringBuilder handleHttpResponse(CloseableHttpResponse response, String backendURL) throws IOException {
+    private static StringBuilder handleHttpResponse(ClassicHttpResponse response, String backendURL)
```

Comment:
Shall we change the return type to string, which will simplify the caller of this method.